scripts/compile_kakarot.py

```diff
@@ -0,0 +1,40 @@
+import logging
+from asyncio import run
+from datetime import datetime
+
+from starknet_py.compile.compiler import Compiler
+
+from scripts.constants import CONTRACTS, SOURCE_DIR
+from scripts.utils.starknet import dump_artifacts
+
+logging.basicConfig()
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+
+async def main():
+    logger.info(f"ℹ️  Compiling contracts")
+
+    artifacts = {}
+    for contract, is_contract_account in [
```

Comment:
Might be good to have this datastructure defined in `scripts.constants`, so it is easier to someone fresh in the codebase to potentially add a contract to the system